code/modules/modular_computers/file_system/programs/airestorer.dm

```diff
@@ -71,12 +71,18 @@
 		ai_slot.locked = FALSE
 		restoring = FALSE
 		return
-	ai_slot.locked =TRUE
+	ai_slot.locked = TRUE
 	A.adjustOxyLoss(-5, 0, FALSE)
 	A.adjustFireLoss(-5, 0, FALSE)
 	A.adjustBruteLoss(-5, 0)
+
+	// Please don't forget to update health, otherwise the below if statements will probably always fail.
+	A.updatehealth()
```

Comment:
```suggestion
	A.adjustOxyLoss(-5, FALSE)
	A.adjustFireLoss(-5, FALSE)
	A.adjustBruteLoss(-5, FALSE)
	A.updatehealth()
```
though, I will note this kinda fix will have to be changed again in #52391 when updateHealth's behavior is adjusted

Reply:
Looks like #52391 totally removes updatehealth() call requirement entirely, which would indirectly fix this.